### ISSUE TITLE
Join equivalent element for Doughnut and Pie charts 

### ISSUE THREAD

**Issue by Kvaz1r** [wxIshiko/wxCharts#63]
Maybe should join equivalent(in string key and in color) elements into one part instead creating separate?   
So, for example, this two:
```
	chartData.AppendSlice(wxChartSliceData(40, wxColor(0xB19F94), "Grey"));
	chartData.AppendSlice(wxChartSliceData(140, wxColor(0xB19F94), "Grey")); 
```
will make one Grey piece with value = 180?

**Comment by xleclercq** [comment]
It's a bit of an edge case. I'm not sure it's going to make much difference in practice. Implementation-wise it will force us to iterate through the vector to check if a slice with the same name already exists but the number of slices will always be low so I think that is OK.
I don't have a strong opinion on this. If you want to make that change go ahead.

**Comment by Kvaz1r** [comment]
Yes, for static chart it's really rare case with non-accurate input data, but for dynamic(I want expand opportunity for add/update data for most common charts and this type first of them) it'll be really important.

**Comment by xleclercq** [comment]
Ah yes OK, I understand better now.

**Comment by Kvaz1r** [comment]
Was implemented in #69
